broker/test/test_broker/mocks/cloudController.js

```diff
@@ -180,39 +183,32 @@ function getSpaces(space_guid) {
   return nock(cloudControllerUrl)
     .get('/v2/spaces')
```

Comment:
Since this pull request is for `/v2/spaces/{guid}` and `/v2/organizations/${guid}`, I think it is better to make the change for `/v2/spaces` and `/v2/organizations` in a new PR.